test/uk/gov/hmrc/mobilepayments/common/BaseSpec.scala

```diff
@@ -32,7 +33,8 @@ import scala.concurrent.ExecutionContext
 
 trait BaseSpec extends AnyWordSpec with MockFactory with Matchers with DefaultAwaitTimeout with AuthorisationStub {
   implicit lazy val ec: ExecutionContext = scala.concurrent.ExecutionContext.Implicits.global
-
+  val mockHttp:                   HttpClientV2    = mock[HttpClientV2]
```

Comment:
make mockHttp and mockRequestBuilder as implicit , then you won't have to call them again and again in Connectors